libraries/Microsoft.Bot.Builder.Ai.LUIS/Microsoft.Bot.Builder.Ai.LUIS.csproj

```diff
@@ -18,7 +18,7 @@
 	<PackageId>Microsoft.Bot.Builder.Ai.LUIS</PackageId>
     <Description>LUIS Middleware and Recognizers for the Microsoft Bot Builder SDK</Description>
     <Summary>This library implements C# classes for building bots using LUIS.</Summary>
-    <GeneratePackageOnBuild>true</GeneratePackageOnBuild>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
```

Comment:
Ooops, so yeah this was wrong, but this fix isn't right either. The default for this property is `false`, so you only ever need to add the property with the value of `true` when you _want_ packages to be built. 

What I messed up was that I didn't move the `<GeneratePackageOnBuild>` property into the conditional group:

`<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">`

So that's what we really need to fix here or you're going to end up with no packages for release mode builds either with this.